requirements.txt

```diff
@@ -1,5 +1,6 @@
 setuptools
 requests
 pyyaml
-matplotlib
```

Comment:
Are you QUITE CERTAIN that matplotlib is no longer required?  Please demonstrate how this PR eliminates all possible references to matplotlib.